### ISSUE TITLE
Sponsor detail view should display the logo if availiable and if not fallback to the sponsor name.  

### ISSUE THREAD

**Issue by elmq0022** [chicagopython/chipy.org#423]
Please see attached.  I think we should do the logo if we have it and if not we should use the sponsor's name.  In terms of accessibility the logo should have the company as a fallback.  

![sponsor-example](https://user-images.githubusercontent.com/3450848/142128403-838dbde1-4afe-4cb8-853a-780baf99b758.png)

This should be the correct template to update
chipy_org/apps/sponsors/templates/sponsors/sponsor_detail.html

Also, a little space between the paragraph and the "visit site" link would be nice.  Heck some space between the logo and the paragraph would be great too.

**Comment by elmq0022** [comment]
Also lets make make the description allow basic html tags so proper paragraphs can be added to a description.  We have django-bleach==0.6.1 available so we can filter for any script tags and such.  

There's also a couple rich text editor fields (not sure why we have two) available which would be extra credit / stretch goal.  I'd be really really happy with basic tags for now. Just prevent large blocks of text.

**Comment by elmq0022** [comment]
https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#autoescape

Explains autoescape; turn it off for this section and filter the text with django bleach.
